### PR TITLE
Remove deprecated queries 

### PR DESCRIPTION
**Pull request by TheGallery**
**What**:
Remove `query*` and `get*` queries from the available commands.

**Why**:
These queries have been deprecated in a previous release and are no longer used.

**How**:
Filter the imported queries from `dom-testing-library` to use only the ones that are relevant.

**Checklist**:

- [x] Documentation
- [x] Tests
- [ ] Ready to be merged

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ function configure({fallbackRetryWithoutPreviousSubject, ...config}) {
   return configureDTL(config)
 }
 
-const queryNames = Object.keys(queries)
-
-const deprecatedRegex = /^(get|query)/
+const availableRegex = /^(find|get)/
+const deprecatedRegex = /^(get)/
```
Comment (kentcdodds) [inline src/index.js:9]: Let's just get rid of all deprecated stuff in this release 👍

**Comment by TheGallery** [comment]
I think I got everything now 🕵️ 🤞

**Review by kentcdodds — APPROVED** [review]
Solid! Thank you! When this is merged it should get automatically released as a beta version. Then we can have folks test it out before merging the beta branch into master to get the official release.

**Comment by TheGallery** [comment]
Great! I look forward to contributing more.

**Comment by kentcdodds** [comment]
Thanks!

**Pull request merged**